aggregator/templates/terms.html

```diff
@@ -0,0 +1,57 @@
+{% extends "base.html" %}
+
+{% block title %}API Terms of use{% endblock title %}
+
+
+{% block content %}
+
+<h1>Democracy Club API Terms of use</h1>
+<p>This applies to the Developers API at
+  <a href="https://developers.democracyclub.org.uk">https://developers.democracyclub.org.uk</a></p>
+<p><em>Latest version (2 November 2019)</em></p>
+
+<p>Our API is currently in private beta.</p>
+
+<h2>An API key will be provided <a href="https://democracyclub.org.uk/contact/">on request</a></h2>
+
+<p>Users of the API agree to:</p>
+
+<ul>
+<li>Attribute Democracy Club, ideally with our logo and a link to <a href="https://democracyclub.org.uk">our website</a>;</li>
+<li>Where possible, send Democracy Club a link to whatever is created and any evidence of impact;</li>
+<li>Allow Democracy Club to publicly share the number of API requests made.</li>
+</ul>
+
+<p>There is no fee or rate limiting, however we request notification of large amounts of traffic so we can maintain the quality of service for all users.</p>
+
+<p>If you would like to discuss different terms of service, please get in touch.</p>
+
+<p>Democracy Club reserves the right to withdraw access if the data is used in such a way that:</p>
+<ul>
+<li>misleads voters (for example, an app to show users their nearest polling station, instead of the correct one allocated to them based on their registered address);</li>
+<li>harms Democracy Club’s ability to deliver the data in future (for example, by weakening the trust placed in us by volunteers or electoral administrators);</li>
+<li>harms the perception of Democracy Club as a non-partisan actor;</li>
+<li>harms the perception of the accuracy of Democracy Club’s data;</li>
+<li>breaks the law (e.g. by the use of personal data for purposes other than an activity that supports or promotes democratic engagement, see s.8(e) of the Data Protection Act 2018).  </li>
```

Comment:
Given that twitter chat with Owen B, I wonder whether we remove the e.g. ?

Reply:
I'm still not convinced / don't know enough, and reckon this e.g is alright